pylintrc

```diff
@@ -259,6 +260,9 @@ good-names=i,
            p2,
            G,   # graph (networkx uses G for graph as mathematicians do)
            T,   # generic TypeVar used by mypy
+           to,
+           Schema,
+           id,
```

Comment:
Do we really want this? `id` is also a builtin.

Reply:
Ok, I renamed it to `uuid`